proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/saml/HubResponseTranslator.java

```diff
@@ -87,6 +92,30 @@ public Response translate(HubResponseContainer hubResponseContainer) {
         );
     }
 
+    private void validateHubResponseContainerAttributes(HubResponseContainer hubResponseContainer) {
+        if ( hubResponseContainer == null ) {
+            throw new IllegalArgumentException("HubResponseContainer is null.");
```

Comment:
Would it be better to use a more specific exception? HubResponseException or something like that.

Reply:
Updated to use HubResponseTranslationException